Microsoft.Toolkit.Uwp.UI/Media/RadialGradientBrush.cs

```diff
@@ -57,6 +60,8 @@ public RadialGradientBrush(Color startColor, Color endColor)
         {
             GradientStops.Add(new GradientStop() { Color = startColor, Offset = 0.0 });
             GradientStops.Add(new GradientStop() { Color = endColor, Offset = 1.0 });
+            RegisterAppResumeHandler();
```

Comment:
This is not necessary (same in the below constructor). We are already calling this()

Reply:
Great, I'll remove them.